backend/experiment/migrations/0019_add_consent_field.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.23 on 2024-02-06 09:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('experiment', '0018_set_default_question_data'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='experimentseries',
+            options={'verbose_name_plural': 'Experiment Series'},
+        ),
+        migrations.AddField(
+            model_name='experiment',
+            name='consent',
+            field=models.TextField(blank=True, default=''),
```

Comment:
I wonder whether a FileUpload field would be better?

Reply:
I think a file field will be better. I made a new branch for this and will set this one to draft for the time being.